src/hooks/broadcast.ts

```diff
@@ -55,4 +57,20 @@ const useUpdateBroadcast = (queryClient: QueryClient) =>
     }
   })
 
-export { useBroadcastDashboardQuery, useUpdateBroadcast, usePastBroadcastsQuery }
+const useLookupTemplateRecords = () =>
+  useQuery({
+    queryKey: ['lookupTemplateRecords'],
+    queryFn: getLookupTemplateRecords
+  })
```

Comment:
You should set staleTime to avoid fetching too often, as I believe this data won't change that much.
```suggestion
  useQuery({
    queryKey: ['lookupTemplateRecords'],
    queryFn: getLookupTemplateRecords,
    staleTime: 5 * 60 * 1000,
  })
```